### DISCUSSION TITLE
How to make two or more vehicles occupy the same point

### DISCUSSION THREAD

**Discussion by Monf1a** [openTCS/opentcs#47] [Q&A]
How to make two or more vehicles occupy the same point? I need to get multiple vehicles to the same work place, but in opentcs simulation, after the first car arrives at the work place, the second car can only wait for it to leave before reaching the work place.

**Comment by swltr** [comment]
Moving multiple vehicles to the same point is something that openTCS's traffic management (i.e. the `Scheduler` implementation) is explicitly supposed to prevent. More than one vehicle cannot physically exist at the same position at the same time. What are you actually trying to achieve?
